src/smc-webapp/projects.cjsx

```diff
@@ -502,7 +502,7 @@ class ProjectsStore extends Store
                     return pay
 
     is_deleted: (project_id) =>
-        return !!@getIn(['project_map', project_id, 'deleted'])
+        return !!@getIn(['project_map', project_id, 'deleted']) 
```

Comment:
This part of the diff does not but add a blank space at the end of the line.  Please enable “Strip trailing whitespace: remove whenever file is saved” in CoCalc account settings, then open the file, then save it (with cursor not in live 505).  Then make another commit.